src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -61,7 +61,7 @@ import {
     isCardTransaction as isCardTransactionTransactionUtils,
     isDistanceRequest as isDistanceRequestTransactionUtils,
     isPerDiemRequest as isPerDiemRequestTransactionUtils,
-    isReceiptBeingScanned as isReceiptBeingScannedTransactionUtils,
+    isScanning,
```

Comment:
NAB: `isScanning` seems like a boolean, but it's a function right? I'd prefer it was more obvious that it is a function.

Suggestions: `checkIfScanning` or `getScanningStatus`